### PR TITLE
Adding BigPanda support within detectors.

### PR DESCRIPTION
**Pull request by benscholler**
This PR adds a new notification for BigPanda support. You can see the BigPanda type here:
https://developers.signalfx.com/detectors_reference.html

### CODE REVIEW

File: tests/test_signal_analog_detectors.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from signal_analog.flow import Assign, Data, Detect, Program, Ref, When
 from signal_analog.charts import TimeSeriesChart
 from signal_analog.detectors import EmailNotification, PagerDutyNotification, \
+                                    BigPandaNotificataion, \
```
Comment (dogonthehorizon) [inline tests/test_signal_analog_detectors.py]: There's a typo here that prevents the tests from running successfully :(
https://travis-ci.org/Nike-Inc/signal_analog/jobs/589151209

Comment (benscholler) [reply tests/test_signal_analog_detectors.py]: It should be better now!

Comment (dogonthehorizon) [reply tests/test_signal_analog_detectors.py]: Thanks!

**Comment by dogonthehorizon** [comment]
We'll get this released at some point today as a new minor release :) Thanks again for contribution @benscholler !

**Pull request merged**